osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_subject.txt

```diff
@@ -0,0 +1 @@
+Extraction Order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}": {{ successful_exports_count }} Export{{ successful_exports_count|pluralize }} finished successfully, 1 failed
```

Comment:
I don't think there will always be `1 failed`, so maybe remove that hard coded part :-)